sdk/python/lib/pulumi/resource.py

```diff
@@ -408,8 +410,9 @@ def __init__(self,
         """
 
         # Expose 'merge' again this this object, but this time as an instance method.
-        self.merge = self._merge_instance
-        self.merge.__func__.__doc__ = ResourceOptions.merge.__doc__
+        # https://github.com/python/mypy/issues/2427
```

Comment:
Minor - but for things like this comment - are we allowed to inline this into the `# type: ignore` comment?  If not - perhaps also briefly summarize the issue in the comment here?

Reply:
Not as far as I'm aware. I'll update the format.